src/main/webapp/view/transcript/transcript.js

```diff
@@ -21,6 +21,7 @@ const ENDPOINT_TRANSCRIPT = '/transcript';
  */
 // TODO: Delete this method once TranscriptServlet's doPost()
 // is called in LectureServlet.
+// TODO: Add videoId as a param.
```

Comment:
Why add this if the other TODO says to delete the method?

Reply:
For transcripts to work while this function exists the param needs to be added